docs/security.md

```diff
@@ -0,0 +1,48 @@
+---
+id: security
+title: 8 - Security
+---
+
+When writing Chialisp, security concerns should be at the front of your mind.  The language is specifically designed to secure money on a network with *no centralized authority* to enforce rules.  The only person standing in the way of attackers and potentially large sums of money is going to be you.
+
+## An unsigned solution is an insecure solution
```

Comment:
Well, not always. Announcements can also prevent farmers from morphing solutions. But I think those are the only two things that do.